sql/moz-fx-data-shared-prod/cloudflare_derived/browser_usage_v1/checks.sql

```diff
@@ -0,0 +1,2 @@
+#warn
+{{ is_unique(["dte", "device_type", "user_type", "location", "browser", "operating_system"]) }}
```

Comment:
```suggestion
{{ is_unique(["date", "device_type", "user_type", "location", "browser", "operating_system"]) }}
```

I assume the field is called `date`?

Reply:
Yeah it's dte in all 3 tables